### PR TITLE
Use scalapb 0.8.4, grpc 1.18.0, and dependencies

### PR DESCRIPTION
**Pull request by gseitz**

### CODE REVIEW

**Comment by gseitz** [comment]
Yikes, [V0.17.1 on Mac](https://travis-ci.org/bazelbuild/rules_scala/jobs/493666116) failed because of a network glitch: `Unknown host: central.maven.org` All other builds went green.

File: scala_proto/scala_proto.bzl [hunk 1]
```diff
@@ -233,8 +233,8 @@ def scala_proto_repositories(
 
     native.maven_jar(
         name = "scala_proto_rules_guava",
-        artifact = "com.google.guava:guava:19.0",
-        sha1 = "6ce200f6b23222af3d8abb6b6459e6c44f4bb0e9",
+        artifact = "com.google.guava:guava:25.1-android",
```
Comment (johnynek) [inline scala_proto/scala_proto.bzl]: What’s up with the android suffix here?

**Comment by gseitz** [comment]
See https://github.com/bazelbuild/rules_scala/pull/691/files#diff-c7a717982f375d79e93079335dc26d4cR236

**Comment by johnynek** [comment]
Thanks!

**Pull request merged**